### ISSUE TITLE
HighCharts: Excel Accessibility

### ISSUE THREAD

**Issue by Ewallye** [highcharts/highcharts#13647]
#### Expected behaviour
Generate an Excel spreadsheet that has 1) document properties (e.g. document title, author, subject, keywords/tags, etc.): Allow the configuration the addition of these values.
 and 2) column/row headers specified. Please reference the following: Only type headers into the first row and/or column and identify the header row and/or column by opening the Name Manager (Alt + M, N). Check that each data table corresponds to a Name and that the Name and Range follow the following formats:

For data tables with one row of column headers:

- “ColumnTitle[Name]..D10” and “=Sheet1!$A$1”, where A1 is substituted for the first cell of the table and D10 is substituted for the last cell of the table.

For data tables with one column of row headers:

- “RowTitle[Name]..D10” and “=Sheet1!$A$1”, where A1 is substituted for the first cell of the table and D10 is substituted for the last cell of the table.

For data tables with both one row and one column of headers:

- “Title[Name]..D10” and “=Sheet1!$A$1”, where A1 is substituted for the first cell of the table and D10 is substituted for the last cell of the table.

#### Actual behaviour
Even though there is an accessible HTML table, users may want to also download a table version of the chart in an accessible format as well.

The excel doc provided after the selection of the Download XLS menu item is missing valued document properties.  

Also, the table provided in the excel is not created correctly to assist the screen reader in announcing the column header and table data effectively.

#### Live demo with steps to reproduce

None

#### Product version

https://www.highcharts.com/demo/accessible-line
#### Affected browser(s)
None

**Comment by sebastianbochan** [comment]
Hi @Ewallye,
Thank you for reporting.

FYI @oysteinmoseng
